From: Kefeng Wang <wangkefeng.wang@huawei.com>
Subject: mm: migrate: convert to folio_isolate_movable()
Date: Mon, 26 Aug 2024 12:01:31 +0800

Directly use folio_isolate_movable() helper in isolate_folio_to_list().

Link: https://lkml.kernel.org/r/20240826040132.1202297-4-wangkefeng.wang@huawei.com
Signed-off-by: Kefeng Wang <wangkefeng.wang@huawei.com>
Cc: Baolin Wang <baolin.wang@linux.alibaba.com>
Cc: David Hildenbrand <david@redhat.com>
Cc: Matthew Wilcox (Oracle) <willy@infradead.org>
Cc: Zi Yan <ziy@nvidia.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/migrate.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/mm/migrate.c~mm-migrate-convert-to-folio_isolate_movable
+++ a/mm/migrate.c
@@ -197,8 +197,8 @@ bool isolate_folio_to_list(struct folio
 	if (lru)
 		isolated = folio_isolate_lru(folio);
 	else
-		isolated = isolate_movable_page(&folio->page,
-						ISOLATE_UNEVICTABLE);
+			isolated = folio_isolate_movable(folio,
+							 ISOLATE_UNEVICTABLE);
 
 	if (!isolated)
 		return false;
_
